classes/Form/ConfigurationForm.php

```diff
@@ -159,6 +174,11 @@ public function generate()
         $helper->fields_value['GA_CROSSDOMAIN_ENABLED'] = Configuration::get('GA_CROSSDOMAIN_ENABLED');
         $helper->fields_value['GA_ANONYMIZE_ENABLED'] = Configuration::get('GA_ANONYMIZE_ENABLED');
 
+        $gaCancelledStates = json_decode(Configuration::get('GA_CANCELLED_STATES'));
+        if (!empty($gaCancelledStates) || $gaCancelledStates !== null) {
```

Comment:
```suggestion
        if (!empty($gaCancelledStates)) {
```

Reply:
@PierreRambaud If the value does not exist in database, it returns NULL, so, maybe better be safe?

Reply:
```
<?php

$gaCancelledStates = null;

$isEmpty = empty($gaCancelledStates);

var_dump($isEmpty);
```

Result:
```
bool(true)
```

The second check is redundant with the first one and can be removed.

Reply:
OK, done guys, thanks!